components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m

```diff
@@ -204,6 +210,32 @@ - (void)setInkColor:(UIColor *)inkColor {
       inkColor ?: [[UIColor alloc] initWithWhite:0 alpha:(CGFloat)0.14];
 }
 
+- (void)setRippleColor:(UIColor *)rippleColor {
+  if (_rippleColor == rippleColor || [_rippleColor isEqual:rippleColor]) {
+    return;
+  }
+  _rippleColor = rippleColor;
+
+  // If no ripple color then reset to the default ripple color.
+  self.rippleTouchController.rippleView.rippleColor =
+      rippleColor ?: [[UIColor alloc] initWithWhite:0 alpha:(CGFloat)0.14];
```

Comment:
Please move this color to a static method at the top of the file and move similar initializations to make use of that method.